manuscript/text/03_results.md

```diff
@@ -0,0 +1,24 @@
+# Results
+
+![** tHMM influence diagram for $\bm{N}$ cells with $\bm{K}$ states.** The latent states ($\bm{z}$) represent subpopulation classification of each cell. The observed states ($\bm{x}$) in blue/orange denote measured phenotypes, particularly fitness (i.e. end-of-life fate by binary outcome variable $\phi$, and lifetime by variable $\tau$ in hours). The tHMM is robust to cells that die, such as cell $6$.. ...](./figures/figure2.svg){#fig:tfac}
```

Comment:
(1) Figure captions should be inline with text. (2) Note the position of the `]` here—brackets should encompass all of the caption. (3) Just topic phrase should be bold (enclosed in `**`). (3) Note `figure` folder is lower case.

Reply:
great thank you.
done
@shak360 waiting for your approval

Reply:
Will do Thursday morning, if you can wait. Need to add the first `align` equation that I took out. There is an error with the citation early on in the introduction. We also have the error : `Latex found 4 multiply defined reference(s)` which is causing the build to fail. Can also add the table I think. The bolding in the figure captions is also not working.

Reply:
`fig:tfac` is the reference name used for this figure. Those need to be unique for each figure.